.tekton/basic-component-pipeline.yaml

```
@@ -244,29 +244,186 @@ spec:
       operator: in
       values: [ "true" ]
 
-  - name: apply-tags
+  - name: build-container-s390x
     params:
     - name: IMAGE
-      value: $(tasks.build-container.results.IMAGE_URL)
-    - name: ADDITIONAL_TAGS
+      value: $(params.output-image-repo):$(tasks.determine-image-tag.results.IMAGE_TAG)-s390x
+    - name: DOCKERFILE
+      value: $(params.dockerfile)
+    - name: CONTEXT
+      value: $(params.path-context)
+    - name: HERMETIC
+      value: $(params.hermetic)
+    - name: PREFETCH_INPUT
+      value: $(params.prefetch-input)
+    - name: IMAGE_EXPIRES_AFTER
+      value: $(params.image-expires-after)
+    - name: COMMIT_SHA
+      value: $(tasks.clone-repository.results.commit)
+    - name: BUILD_ARGS
+      value:
+      - VERSIONS_SUFFIX=$(params.output-tag-suffix)
+      - MAIN_IMAGE_TAG=$(tasks.determine-image-tag.results.IMAGE_TAG)
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+    - name: CACHI2_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
+    - name: PLATFORM
+      value: linux/s390x
+    taskRef:
+      params:
+      - name: name
+        value: buildah-remote-oci-ta
+      - name: bundle
+        value: quay.io/redhat-appstudio-tekton-catalog/task-buildah-remote-oci-ta:0.1@sha256:7f07fd3e10eaeec2835880bd3cb12e73dca42e4bb3fa15ffac1b8bfcd53df435
```

[optional]
Looks like there's more fresh version of the task already although it does not change anything significant for multi-arch builds. Only **if you have extra cycles**, you can try use that so this PR goes out with the latest and greatest.

```suggestion
        value: quay.io/redhat-appstudio-tekton-catalog/task-buildah-remote-oci-ta:0.2@sha256:6fa1e7972e19c5bd8cc26c29c4de6ddb7f4f538dca10465bc4c1cb5a16400886
```

This requires some upgrade steps https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-remote-oci-ta/0.2/MIGRATION.md

This will also require upgrading non-remote `buildah-oci-ta` also to 0.2, but the migration steps are the same. https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-oci-ta/0.2/MIGRATION.md
Image reference for that: `quay.io/redhat-appstudio-tekton-catalog/task-buildah-oci-ta:0.2@sha256:812f2886915b2bd5ea306c179bb778703d47c95585a93953246439ffe3d2ca04`.